Source/WTF/Scripts/Preferences/WebPreferencesInternal.yaml

```diff
@@ -608,13 +608,15 @@ LegacyOverflowScrollingTouchEnabled:
 LineHeightUnitsEnabled:
   type: bool
   humanReadableName: "lh / rlh units"
-  humanReadableDescription: "Enable the lh and lhr units"
+  humanReadableDescription: "Enable the lh and rlh units"
   webcoreBinding: DeprecatedGlobalSettings
   defaultValue:
     WebKitLegacy:
-      default: false
+      default: true
     WebKit:
-      default: false
+      default: true
+    WebCore:
+      default: true
```

Comment:
Shouldn't this have also modified `m_lineHeightUnitsEnabled` in `page/DeprecatedGlobalSettings.h` and `LineHeightUnitsEnabled` in `Tools/DumpRenderTree/TestOptions.cpp`?

Reply:
It's also unclear what `parseUnit()` in `css/typedom/CSSUnitValue.cpp` should say